playbook/app/pb_kits/playbook/pb_table/_table.tsx

```diff
@@ -85,33 +88,92 @@ const Table = (props: TableProps): React.ReactElement => {
         className
     )
 
+    useEffect(() => {
+        const handleStickyColumns = () => {
+            let accumulatedWidth = 0;
+
+            stickyLeftcolumn.forEach((colId) => {
+                const header = document.querySelector(`th[id="${colId}"]`);
+                const cells = document.querySelectorAll(`td[id="${colId}"]`);
+
+                if (header) {
+                    header.classList.add('sticky');
+                    header.style.left = `${accumulatedWidth}px`;
+
+                    accumulatedWidth += header.offsetWidth;
+                }
+
+                cells.forEach((cell) => {
+                    cell.classList.add('sticky');
+                    cell.style.left = `${accumulatedWidth - header.offsetWidth}px`;
+                });
+            });
+        };
+
+        setTimeout(() => {
+            handleStickyColumns();
+        }, 50);
+
+        window.addEventListener('resize', handleStickyColumns);
+
+        return () => {
+            window.removeEventListener('resize', handleStickyColumns);
+        };
+    }, [stickyLeftcolumn]);
+
     useEffect(() => {
         const instance = new PbTable()
         instance.connect()
     }, [])
 
     return (
         <>
-            {isTableTag ? (
-                <table
-                    {...ariaProps}
-                    {...dataProps}
-                    {...htmlProps}
-                    className={classNames}
-                    id={id}
-                >
-                    {children}
-                </table>
-            ) : (
-                <div
-                    {...ariaProps}
-                    {...dataProps}
-                    {...htmlProps}
-                    className={classNames}
-                    id={id}
-                >
-                    {children}
+            {responsive === 'scroll' ? (
+                <div className='table-responsive-scroll'>
```

Comment:
Let's try something. Let's try converting the react fragment on line 130 to a div, and string interpolating the conditional className. 

So it would end up like this:
```
<div className={responsive === 'scroll' ? 'table-responsive-scroll' : ''}>
```

Reply:
Hey Jasper! Thanks for the suggestion, i tried this but ended up adding a new wrapper div around out kit permanently, the  fragments werent adding anything to the dom.